RELEASE_NOTES.md

```diff
@@ -1,9 +1,39 @@
 # What's new in YugabyteDB Voyager
+
 #### New features, key enhancements, and bug fixes
+
 Included here are the release notes for the [YugabyteDB Voyager](https://docs.yugabyte.com/preview/migrate/) v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.4 - June 30, 2023
+
+#### Key enhancements:
```

Comment:
@priyanshi-yb Can you use the release notes that finally got published on https://docs.yugabyte.com/preview/migrate/release-notes/?

There are some subtle changes, including terminology and grammar. Would be good to be consistent